src/core/libmaven/mavenparameters.cpp

```diff
@@ -240,8 +240,15 @@ void MavenParameters::setOptionsDialogSettings(const char* key, const char* valu
         if(indexOfIonizationMode==1) ionizationMode=0;
         else if(indexOfIonizationMode==2) ionizationMode=+1;
         else if(indexOfIonizationMode==3) ionizationMode=-1;
-        // set auto to zero
-        else ionizationMode=0;
```

Comment:
We can just call the setIonizationMode() function here, right?

Reply:
@shubhra-agrawal Requested change is made!

Reply:
@IpankajI  @shubhra-agrawal  should we not use setIonizationMode() for handling all the cases? why just auto-detect?  Imo the function should be modified to handle all the cases.